app/models/terms_of_service.rb

```diff
@@ -26,6 +27,10 @@ class TermsOfService < ApplicationRecord
 
   NOTIFICATION_ACTIVITY_CUTOFF = 1.year.freeze
 
+  def self.current
+    TermsOfService.live.first || TermsOfService.upcoming.first # For the case when none of the published terms have become effective yet
```

Comment:
Do these need the explicit class?

Reply:
:man_facepalming: 

In my defence, it is getting rather hot around here :hot_face: 

Thanks for catching this!